cmd/che/che_harness_test.go

```diff
@@ -121,6 +121,8 @@ func registerCheFlags(flags *flag.FlagSet) {
 	flags.StringVar(&CfgInstance.artifactsDir, "artifacts-dir", "/tmp/artifacts", "If is specified test harness will save all reports in the given directory, if not will save artifacts in the current directory. Default dir is /tmp/artifacts")
 	flags.StringVar(&CfgInstance.metricsFiles, "metrics-files", "", "If it is set che test harness start to send the data to AWS S3. You should have valid secrets in the files")
 	flags.StringVar(&metadata.Namespace.Name, "che-namespace", "eclipse-che", "Namespace where che-operator was deployed before launch tests. Default namespace is `eclipse-che`")
+	flags.StringVar(&metadata.User.Username, "username", "admin", "Username of user to log in. Default username is `admin`")
```

Comment:
Why we need this flags here?

Reply:
This change is related to [this](https://github.com/che-incubator/che-test-harness/pull/9/files#diff-37a10a603d9d23f59c724e83815fdad8R88) change. I think it should be possible to default those values to "admin" - so it will not be needed to be set here. I just liked the idea of simply parametrizing it. But if you think it's over-engineering, I can remove that. :)